src/components/filter/DateFilter.tsx

```diff
@@ -88,7 +119,7 @@ export const DateFilter = ({ containerStyle, data, filters, setFilters }: Props)
 
   return (
     <>
-      <Label>{texts.filter.date}</Label>
+      <Label bold>{texts.filter.date}</Label>
       <WrapperRow spaceBetween>
         {data.map((item) => {
           useEffect(() => {
```

Comment:
in this effect i found the following code

```
const endTimeOfDay = item.name === 'start_date' ? 'T00:00:00+01:00' : 'T22:59:59+01:00';
```

if 22:59+1 = 23:59 than 00:00+01 = 01:00, isn't it? why did we do that? shouldn't it be 00:00+00 and 23:59+00 to have the full time range of a day?

Reply:
The reason we did this is that we needed to add this data in the date filter we developed for SUE as per api requirements.

If we select a date, it automatically adds, for example, `dateT14:23:43+01:93`. If we added this value, we could not filter correctly.

Reply:
and what do you say about

> 00:00+01 = 01:00

?
doesn't we want from 00?